x/observer/client/cli/tx_disable_cctx_flags.go

```diff
@@ -0,0 +1,43 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func CmdDisableCCTXFlags() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "disable-cctx-flags [disable-inbound] [disable-outbound]",
+		Short: "Disable inbound and outbound cross-chain flags",
```

Comment:
Similar to another reply, this tries to retain the existing functionality so that we can disable only one if needed. 
The main reason for splitting the two messages is to have one authorized policy per message type